src/olympia/addons/templates/addons/impala/details.html

```diff
@@ -18,130 +18,258 @@
   <meta property="og:description" content="{{ addon.summary|striptags }}">
 {% endblock %}
 
-{% block content %}
-{{ impala_breadcrumbs([(addon.type_url(), amo.ADDON_TYPES[addon.type]),
-                       (None, addon.name)]) }}
+{% block header_content %}
 <div itemscope itemtype="http://schema.org/WebApplication">
   <link itemprop="SoftwareApplicationCategory" href="http://schema.org/OtherApplication" />
   <aside class="secondary addon-vitals">
 
-  {# This assumes we'll never charge for add-ons. Ha. #}
-  <span itemprop="offers" itemscope itemtype="http://schema.org/Offer">
-    <meta itemprop="price" content="$0" />
-    <meta itemprop="priceCurrency" content="USD" />
-    <link itemprop="availability" href="http://schema.org/InStock" />
-  </span>
-
-  <div itemprop="aggregateRating" itemscope="" itemtype="http://schema.org/AggregateRating">
-    <meta content="{{ addon.average_rating }}" itemprop="ratingValue">
-    {{ addon.average_rating|stars(large=True) }}
-
-    <div>
-      <a id="reviews-link" href="{{ addon.reviews_url }}">
-      {% trans cnt=addon.total_reviews, num=addon.total_reviews|numberfmt %}
-        <span itemprop="ratingCount">{{ num }}</span> user review
-      {% pluralize %}
-        <span itemprop="ratingCount">{{ num }}</span> user reviews
-      {% endtrans %}
-      </a>
+    {# This assumes we'll never charge for add-ons. Ha. #}
+    <span itemprop="offers" itemscope itemtype="http://schema.org/Offer">
+      <meta itemprop="price" content="$0" />
+      <meta itemprop="priceCurrency" content="USD" />
+      <link itemprop="availability" href="http://schema.org/InStock" />
+    </span>
```

Comment:
At least let's remove the `$` in `$0` since that is duplicated information with `priceCurrency` already describing the currency?

But on the other hand, let's not kill any more baby dragons.